media-fonts/font-misc-misc/font-misc-misc-1.1.2-r2.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit xorg-3
+
+DESCRIPTION="X.Org miscellaneous fonts"
+
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
+
+BDEPEND="x11-apps/bdftopcf"
+
+DOCS=( README ChangeLog )
```

Comment:
Same comment throughout the series. I don't want to install these files. They're really not useful. `HOMEPAGE` links to the git repo that contains all this information.